src/net_processing.cpp

```diff
@@ -1128,6 +1143,9 @@ PeerLogicValidation::PeerLogicValidation(CConnman* connmanIn, BanMan* banman, CS
     // timer.
     static_assert(EXTRA_PEER_CHECK_INTERVAL < STALE_CHECK_INTERVAL, "peer eviction timer should be less than stale tip check timer");
     scheduler.scheduleEvery([this, consensusParams] { this->CheckForStaleTipAndEvictPeers(consensusParams); }, std::chrono::seconds{EXTRA_PEER_CHECK_INTERVAL});
+
+    const std::chrono::milliseconds delta = std::chrono::milliseconds(10 * 60 * 1000 + GetRand(5 * 60 * 1000));
```

Comment:
The same std::chrono value used inside ReattemptInitialBroadcast, maybe this justifies making a constant?

Reply:
Oh I see you removed REATTEMPT_BROADCAST_INTERVAL, well. Feel free to close this one then.